pypxlib/__init__.py

```diff
@@ -180,7 +196,12 @@ def _serialize_to(self, value, pxval_value):
 
 class BytesField(Field):
 	def _deserialize(self, pxval_value):
-		return pxval_value.str.val[:pxval_value.str.len]
+		try:
+			# Legacy support
```

Comment:
As I don't know if `pxval_value.str.val[:pxval_value.str.len]` works for other users, I let there the code with a try-catch block. Maybe it's useless now.